### ISSUE TITLE
Read error from stream: End of file

### ISSUE THREAD

**Issue by curtishall** [bluecherrydvr/bluecherry-apps-issues#51]
On Axis cameras connected to .134:

```
Sep 13 09:34:51 1204-unstable bc-server[22958]: E(8/Axis 210A 1): Read error from stream: End of file
Sep 13 09:34:51 1204-unstable bc-server[22958]: E(9/Axis 210A 2): Read error from stream: End of file
Sep 13 09:34:51 1204-unstable bc-server[22958]: E(6/Brickcom OB-100Ae): Read error from stream: End of file
Sep 13 09:34:51 1204-unstable bc-server[22958]: E(5/Ubiquiti): Read error from stream: End of file
Sep 13 09:34:52 1204-unstable bc-server[22958]: E(10/ACTi ACM-5611): Failed to analyze input stream. Error: -1 (Operation not permitted)
Sep 13 09:34:52 1204-unstable bc-server[22958]: E(10/ACTi ACM-5611): Error starting device stream: 
Sep 13 09:35:01 1204-unstable bc-server[22958]: I(9/Axis 210A 2): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:35:01 1204-unstable bc-server[22958]: I(8/Axis 210A 1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:35:03 1204-unstable bc-server[22958]: I(5/Ubiquiti): Stream started: Video: h264 (Baseline), yuvj420p(pc), 1280x720 [SAR 1:1 DAR 16:9], 1/90000(s) 1/30(c)
Sep 13 09:35:03 1204-unstable bc-server[22958]: B(5/Ubiquiti): motion_flag_consumer has a newer packet than target buffer
Sep 13 09:35:03 1204-unstable bc-server[22958]: I(6/Brickcom OB-100Ae): Stream started: Video: h264 (Baseline), yuv420p, 640x400, 1/90000(s) 1/180000(c)
Sep 13 09:35:42 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 13 09:35:45 1204-unstable bc-server[22958]: E(9/Axis 210A 2): Read error from stream: End of file
Sep 13 09:35:45 1204-unstable bc-server[22958]: E(8/Axis 210A 1): Read error from stream: End of file
Sep 13 09:35:45 1204-unstable bc-server[22958]: E(5/Ubiquiti): Read error from stream: End of file
Sep 13 09:35:45 1204-unstable bc-server[22958]: E(6/Brickcom OB-100Ae): Read error from stream: End of file
Sep 13 09:35:46 1204-unstable bc-server[22958]: E(10/ACTi ACM-5611): Failed to analyze input stream. Error: -1 (Operation not permitted)
Sep 13 09:35:55 1204-unstable bc-server[22958]: I(9/Axis 210A 2): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:35:55 1204-unstable bc-server[22958]: I(8/Axis 210A 1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:35:57 1204-unstable bc-server[22958]: I(5/Ubiquiti): Stream started: Video: h264 (Baseline), yuvj420p(pc), 1280x720 [SAR 1:1 DAR 16:9], 1/90000(s) 1/30(c)
Sep 13 09:35:57 1204-unstable bc-server[22958]: I(6/Brickcom OB-100Ae): Stream started: Video: h264 (Baseline), yuv420p, 640x400, 1/90000(s) 1/180000(c)
Sep 13 09:36:39 1204-unstable bc-server[22958]: E(6/Brickcom OB-100Ae): Read error from stream: End of file
Sep 13 09:36:39 1204-unstable bc-server[22958]: E(8/Axis 210A 1): Read error from stream: End of file
Sep 13 09:36:39 1204-unstable bc-server[22958]: E(5/Ubiquiti): Read error from stream: End of file
Sep 13 09:36:39 1204-unstable bc-server[22958]: E(9/Axis 210A 2): Read error from stream: End of file
Sep 13 09:36:49 1204-unstable bc-server[22958]: I(8/Axis 210A 1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:36:49 1204-unstable bc-server[22958]: I(9/Axis 210A 2): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/100(c)
Sep 13 09:36:50 1204-unstable bc-server[22958]: I(5/Ubiquiti): Stream started: Video: h264 (Baseline), yuvj420p(pc), 1280x720 [SAR 1:1 DAR 16:9], 1/90000(s) 1/30(c)
Sep 13 09:36:51 1204-unstable bc-server[22958]: I(6/Brickcom OB-100Ae): Stream started: Video: h264 (Baseline), yuv420p, 640x400, 1/90000(s) 1/180000(c)
Sep 13 09:37:00 1204-unstable bc-server[22958]: B(6/Brickcom OB-100Ae): motion_flag_consumer has a newer packet than target buffer
```

**Comment by ghost** [comment]
From log i believe this happens due to physical connection problems, which happens sometimes in that network as i know.
You can see that a lot of EOF errors happen on different cameras at nearly same time, followed by a lot of failures to establish connection, followed by media stream connection recovery.
@curtishall may i close this ticket for now?

```
root@1204-unstable:~# egrep 'End|Input/output error' /var/log/bluecherry.log.1 -A 5
Sep 17 06:46:48 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 17 06:46:58 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 17 08:43:20 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 08:43:35 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 17 09:13:23 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 14 times!
Sep 17 09:13:23 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 09:13:57 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 17 09:13:57 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 17 09:14:07 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 09:14:35 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 17 09:14:35 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 17 09:30:10 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 09:30:25 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 17 10:09:46 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 10:10:02 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 17 12:53:19 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 12:53:35 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 17 14:47:22 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 13 times!
Sep 17 14:47:22 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 14:47:38 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 17 14:47:38 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 17 15:09:51 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 2994 times!
Sep 17 15:09:51 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Last message repeated 63 times!
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Read error from stream: Connection timed out
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 1 times!
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Read error from stream: End of file
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(4/Vivotek): Last message repeated 170 times!
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(4/Vivotek): Read error from stream: End of file
Sep 17 15:09:53 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Error starting device stream: 
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Error starting device stream: 
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(4/Vivotek): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:06 1204-unstable bc-server[22958]: E(4/Vivotek): Error starting device stream: 
Sep 17 15:10:07 1204-unstable bc-server[22958]: E(10/ACTi ACM-5611): Last message repeated 11 times!
Sep 17 15:10:07 1204-unstable bc-server[22958]: E(10/ACTi ACM-5611): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:09 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:09 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:09 1204-unstable bc-server[22958]: E(4/Vivotek): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:13 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:13 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Error starting device stream: 
Sep 17 15:10:14 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:14 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Error starting device stream: 
Sep 17 15:10:17 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:17 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Failed to open stream. Error: -5 (Input/output error)
Sep 17 15:10:47 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 3 times!
Sep 17 15:10:47 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 17 15:10:58 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 2 times!
Sep 17 15:10:58 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Stream started after failures
Sep 17 15:10:58 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
--
Sep 17 15:11:31 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 17 15:11:42 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Last message repeated 6 times!
Sep 17 15:11:42 1204-unstable bc-server[22958]: E(1/Grandstream GXV3615): Stream started after failures
Sep 17 15:11:42 1204-unstable bc-server[22958]: I(1/Grandstream GXV3615): Stream started: Video: h264 (Constrained Baseline), yuv420p, 720x480, 1/90000(s) 1/180000(c)
Sep 17 15:11:44 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Last message repeated 6 times!
Sep 17 15:11:44 1204-unstable bc-server[22958]: E(3/ACTi ACM-8211): Stream started after failures
--
Sep 17 19:14:03 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 19:14:45 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 17 19:44:25 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 17 23:34:03 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 5 times!
Sep 17 23:34:03 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 17 23:34:19 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 17 23:34:19 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 17 23:50:25 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 3038 times!
Sep 17 23:50:25 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 17 23:50:35 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 00:00:53 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 18 00:01:09 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 18 00:25:53 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 205 times!
Sep 18 00:25:53 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 00:26:03 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 00:26:34 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 1 times!
Sep 18 00:26:34 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 00:26:44 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 00:28:55 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 11 times!
Sep 18 00:28:55 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 00:29:06 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 00:30:18 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 18 00:50:36 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 125 times!
Sep 18 00:50:36 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 00:50:46 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 01:03:44 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:04:18 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:04:28 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:04:48 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:07:28 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:08:08 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:08:20 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:08:36 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:09:05 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:09:21 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:09:41 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:10:00 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:10:26 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:10:42 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:11:07 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:11:23 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:12:18 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:12:37 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:13:28 1204-unstable bc-server[22958]: E(7/Arecont Vision 5MP): Read error from stream: End of file
Sep 18 01:13:44 1204-unstable bc-server[22958]: I(7/Arecont Vision 5MP): Stream started: Video: h264 (Baseline), yuv420p, 1280x800, 1/90000(s) 1/180000(c)
Sep 18 01:33:54 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 251 times!
Sep 18 01:33:54 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 01:34:04 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 02:28:23 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 9 times!
Sep 18 02:28:23 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 18 02:28:39 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 18 02:36:40 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 18 03:37:39 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 733 times!
Sep 18 03:37:39 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 03:37:49 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 03:55:39 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 103 times!
Sep 18 03:55:39 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 03:55:49 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 04:14:15 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 107 times!
Sep 18 04:14:15 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 04:14:25 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 04:38:24 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Last message repeated 3 times!
Sep 18 04:38:24 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 18 04:38:39 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 18 04:55:44 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 18 05:25:22 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 412 times!
Sep 18 05:25:22 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 05:25:25 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 18 05:25:32 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 05:25:41 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 18 05:53:14 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 163 times!
Sep 18 05:53:14 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 05:53:24 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
Sep 18 06:07:31 1204-unstable bc-server[22958]: E(17/axis-00408c8af0c1): Read error from stream: End of file
Sep 18 06:07:46 1204-unstable bc-server[22958]: I(17/axis-00408c8af0c1): Stream started: Video: mpeg4 (Advanced Simple Profile), yuv420p, 640x480 [SAR 1:1 DAR 4:3], 1/90000(s) 1/1000(c)
Sep 18 06:11:13 1204-unstable bc-server[22958]: B(17/axis-00408c8af0c1): motion_flag_consumer has a newer packet than target buffer
Sep 18 06:25:21 1204-unstable bc-server[22958]: E(12/Front camera demo): Last message repeated 188 times!
Sep 18 06:25:21 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -5 (Input/output error)
Sep 18 06:25:31 1204-unstable bc-server[22958]: E(12/Front camera demo): Failed to open stream. Error: -1094995529 (Invalid data found when processing input)
```

**Comment by curtishall** [comment]
No, this needs more investigation to determine what the problem is.  The network is fine, it's all connected via 100mbit / 1GB copper to a POE switch.  

The only thing that I can think of is the cameras are streaming to .132 also, however the camera should be able to handle 3-5 streams at the same time.

**Comment by ghost** [comment]
i'm enabling debug level logging on .134.

**Comment by curtishall** [comment]
Thanks!

**Comment by ghost** [comment]
Resolved by commits b3df1dbadb26a5c811180ac105b17fadfcd874af, 10bccc2e5b4fec6e5cf229d98047d7e6f0e10d9d. Thanks to Anton. Should be released in the next release after 2.3.13.
